docs/docs/articles/github-actions.md

```diff
@@ -1,135 +1,85 @@
-
 ## Testkube GitHub Action
 
-The testkube GitHub Action is available here <https://github.com/kubeshop/testkube-docker-action> and it enables running the Testkube CLI commands in a GitHub workflow.
+The Testkube GitHub Action installs Testkube and enables running any [Testkube CLI](https://docs.testkube.io/cli/testkube) command in a GitHub workflow. It is available on Github Marketplace <https://github.com/marketplace/actions/testkube-action>.
+The action provides a flexible way to work with your pipeline and can be used with Testkube Cloud or self-hosted platform (self-hosted platform means a k8s cluster that uses an open source Testkube solution (OSS), not Cloud).
 
-The following example shows how to create a test using the GitHub action; a more complex example can be found [here](https://github.com/kubeshop/helm-charts/blob/59054b87f83f890f4f62cf966ac63fd7e46de336/.github/workflows/testkube-docker-action.yaml).
+The following example shows how to create and run a test using the GitHub action on the [Teskube cloud](https://cloud.testkube.io/) instance. Please note that there are no additional steps needed to connect to the k8s cluster as all the necessary data are provided as inputs:
 
 ```yaml
-# Creating test
-- name: Create test
-  id: create_test
-  uses: kubeshop/testkube-docker-action@v1
-  with:
-    command: create
-    resource: test
-    namespace: testkube
-    parameters: "--type k6/script --name testkube-github-action"
-    stdin: "import http from 'k6/http';\nimport { sleep,check } from 'k6';\n\nexport default function () {\n  const baseURI = `${__ENV.TESTKUBE_HOMEPAGE_URI || 'https://testkube.kubeshop.io'}`\n  check(http.get(`${baseURI}/`), {\n    'check testkube homepage home page': (r) =>\n      r.body.includes('Your Friendly Cloud-Native Testing Framework for Kubernetes'),\n  });\n\n\n  sleep(1);\n}\n"
+steps:
+  # Setup Testkube
+  - uses: kubeshop/setup-testkube@v1
+    with:
+      organization: ${{ secrets.TkOrganization }}
+      environment: ${{ secrets.TkEnvironment }}
+      token: ${{ secrets.TkToken }}
+
+  # Use CLI with a shell script
+  - run: |
+      testkube create test --name some-test-name --file path_to_file.json
+      testkube run test some-test-name -f
 ```
 
-## Configuring Your GH Actions for Access to GKE
-
-To obtain set up access to a GKE (Google Kubernetes Engine) from GH (GitHub) actions, please visit the official documentation from GH: <https://docs.github.com/en/actions/deployment/deploying-to-google-kubernetes-engine>.
-
-1. Create a Service Account (SA).
-2. Save it into GH's **Secrets** of the repository.
-3. Run either `Helm` or `Kubectl kubtest` commands against the set up GKE cluster.
-
-## Main GH Action Section Configuration
-
-To install on Linux or MacOS, run:
-
+Create and run tests on self-hosted platform. Please mind that it requires establishing connection with the k8s cluster:
```

Comment:
* It should be a separate section, i.e. `Self-Hosted Testkube OSS`
* I'm not sure if we should show step for AWS without explaining. Every Cloud provider will require different steps before
* Same as above - either secrets should be described, or it should not use secrets

Reply:
made alterations